cloudbuild.yaml

```diff
@@ -8,7 +8,7 @@ steps:
   - name: 'maven:3-jdk-11'
     id: 'Set version for Maven'
     entrypoint: mvn
-    args: [ 'versions:set', '-DnewVersion=${_MAJOR_MINOR_VERSION}.$SHORT_SHA' ]
+    args: [ 'versions:set', '-DnewVersion=${TAG_NAME}' ]
```

Comment:
My only concern with this is whether it makes it easier to build a pipeline that doesn't match the branch it is on. Pipeline5 is the only project (to my knowledge) that we build and release from multiple branches concurrently.

Not a question of the correctness of what you've done here technically, more a strategic question.

Reply:
Will discuss this with the Amsterdam people

Reply:
Yeah agreed. I think that's a good way to move forward without causing too much extra work.